aiida/backends/tests/cmdline/commands/test_process.py

```diff
@@ -219,6 +220,58 @@ def test_list(self):
             self.assertEqual(len(get_result_lines(result)), 1)
             self.assertEqual(get_result_lines(result)[0], str(self.calcs[0].pk))
 
+    def test_list_worker_slot_warning(self):
+        """
+        Test that the if the number of used worker process slots exceeds a threshold,
+        that the warning message is displayed to the user when running `verdi process list`
+        """
+        from aiida.engine import ProcessState
+
+        # Create 6 WorkFunctionNodes (one for each ProcessState)
+        # Do this 28 times such that we have an additional 168 nodes, of which 84 are 'active' states
+        # Total now is 90 active nodes and 90 inactive nodes.
```

Comment:
You create 84 active but the total now is 90??? Also, since we only care about active states, why not just create however many you need with `Running`. No need for the 'complex' logic in the creation of those nodes.

Edit: I see now why there are 90 total, the other 6 come from the `setupClass`. Maybe add this in the comment